sql/src/main/java/io/crate/planner/node/dql/MergePhase.java

```diff
@@ -60,16 +62,31 @@
      */
     @Nullable private PositionalOrderBy positionalOrderBy;
 
+    /**
+     * Creates a MergePhase to combine the results from multiple node operations.
+     * @param jobId The JobID of the entire execution.
+     * @param executionNodeId A unique execution id for this phase.
+     * @param name The name of the MergePhase.
+     * @param numUpstreams The number of nodes to expect data from.
+     * @param numInputs The number of different inputs to read data from.
```

Comment:
Maybe clarify that this is the number of upstream phases? "different inputs" is a bit vague